mcs/class/System.Net.Http/HttpClientHandler.SocketsHandler.Android.cs

```diff
@@ -0,0 +1,20 @@
+namespace System.Net.Http
```

Comment:
This file is a copy of [HttpClientHandler.SocketsHandler.cs](https://github.com/mono/mono/blob/master/mcs/class/System.Net.Http/HttpClientHandler.SocketsHandler.cs) (used for Android before this PR)